packages/api/src/command/medical/cq-directory/search-cq-directory.ts

```diff
@@ -35,16 +38,22 @@ export async function searchNearbyCQOrganizations({
   radiusInMiles?: number;
 }): Promise<CQOrgBasicDetails[]> {
   const radiusInMeters = convert(radiusInMiles).from("mi").to("m");
-
   const patient = await getPatientOrFail({ id: patientId, cxId });
-  const coordinates = await geocodeAddresses(patient.data.address);
+  const addresses = await upsertGeographicCoordinates({
+    patient,
+    reportRelevance: true,
+    updatePatientAddresses: true,
+  });
+
+  const coordinates = addresses.flatMap(address => address.coordinates ?? []);
+  if (!coordinates.length) throw new Error("Failed to get patient coordinates");
 
   const orgs = await searchCQDirectoriesByRadius({
     coordinates,
     radiusInMeters,
   });
 
-  return orgs.map(toBasicOrgAttributes);
+  return uniq(orgs.map(toBasicOrgAttributes));
```

Comment:
Ah, I just realized uniq will not take care of that, since these orgs will be separate objects in memory. I'll switch this to `uniqBy` and use the oid for comparison